### DISCUSSION TITLE
Tarchetypes missing from CRAN

### DISCUSSION THREAD

**Discussion by ghost** [ropensci/tarchetypes#47] [Help]
Hello,

My work only permits packages from CRAN and recently tarchetypes seems to have dropped off.

https://CRAN.R-project.org/package=tarchetypes 

Are there any plans for resolving this?

Thanks,

Marc

**Comment by wlandau** [comment]
`tarchetypes` was taken down when `targets` was archived. (Details at https://github.com/llrs/cran_histories/issues/10.) I resubmitted `tarchetypes` earlier this week, and it is still in the queue to be reviewed. Hopefully it will be back up soon.

**Reply by wlandau** [reply]
`tarchetypes` is now back on CRAN.
